006-securing-kafka-connect-rest-endpoint.md

```diff
@@ -0,0 +1,197 @@
+# Securing the Kafka Connect REST API endpoint
+
+## Current situation
+
+Currently, instances of Kafka Connect that are deployed by the Strimzi operators are configured with the default REST API endpoint settings. This means that the Kafka Connect REST API endpoint uses HTTP on port 8083 and that the Strimzi KafkaConnect, KafkaConnector and KafkaMirrorMaker2 operators make unsecured REST client calls based on this default configuration.
+
+The default network policies created by the Strimzi operators restrict incoming REST API calls to only allow access from the operator pod. If required, further network policies can be provided to override the default policy and allow wider access.
```

Comment:
It might be worth mentioning that at this point, this is not the intention but a side effect of how Network Policies work and how they are discoverable.